pkg/constants/common.go

```diff
@@ -10,13 +10,17 @@ const RepoManagerHost = Prefix + "repo-manager"
 const AppManagerHost = Prefix + "app-manager"
 const RuntimeEnvManagerHost = Prefix + "runtime-env-manager"
 const ClusterManagerHost = Prefix + "cluster-manager"
+const JobManagerHost = Prefix + "job-manager"
+const TaskManagerHost = Prefix + "task-manager"
 const DbCtrlHost = Prefix + "db-ctrl"
 const ApiGatewayPort = "9100" // 91 is similar as Pi, Open[Pi]trix
 const RepoManagerPort = "9101"
 const AppManagerPort = "9102"
 const RuntimeEnvManagerPort = "9103"
 const ClusterManagerPort = "9104"
 const DbCtrlPort = "9105"
+const JobManagerPort = "9106"
+const TaskManagerPort = "9107"
```

Comment:
why port is string type?

and use group block is better.

and the `Prefix` should not be public api, `"openpitrix-app-manager"` is better than `Prefix + "app-manager"`

Reply:
What do you mean by "group block". 
I will change `Prefix` to `prefix`. 
I think `Prefix + "app-manager"` is ok, through we will not change `openpitrix` prefix.

Reply:
I will refactor `constants` later, so this PR keep `constants` not changed